views/make_a_peep.erb

```diff
@@ -0,0 +1,9 @@
+<h1> Make a new peep! </h1>
+ <br>
+ <form>
```

Comment:
if you add your action and post method inside form you can set up route to the next page:
`< form action="/make_a_peep" method ="post" >